Wire-iOS/Sources/Authentication/Backup/BackupRestoreController.swift

```diff
@@ -30,6 +30,7 @@ protocol BackupRestoreControllerDelegate: class {
 
 final class BackupRestoreController: NSObject {
     static let WireBackupUTI = "com.wire.backup-ios"
+    static let WireBackupAdditionalUTI = "com.wire.backup-ios-additional"
```

Comment:
We can create a static array like 
```suggestion
    static let WireBackupUTIs = ["com.wire.backup-ios", "com.wire.backup-ios-additional"]
```